src/styles/blocks/page.scss

```diff
@@ -0,0 +1,19 @@
+$cardWidth: 200px;
+$gapX: 50px;
+$gapY: 40px;
+$paddingX: 40px;
+$paddingY: 50px;
```

Comment:
move variables to separate file _variables.scss and https://github.com/import it in main.scss